src/apps/curve/positions.ts

```diff
@@ -44,12 +47,12 @@ export async function getAllCurvePools(
     return []
   }
   const { data } = await got
-    .get(`https://api.curve.fi/v1/getPools/${blockchainId}/factory`)
+    .get(`https://api.curve.fi/v1/getPools/all/${blockchainId}`)
     .json<CurveApiResponse>()
 
   return data.poolData.map((poolInfo) => ({
-    address: poolInfo.address,
-    size: poolInfo.implementation === 'plain3basic' ? 3 : 2,
+    address: poolInfo.lpTokenAddress,
```

Comment:
Some pools have a different lp token address, and here we really want lp tokens.